Pod/Classes/Forms/UIView+Lookup.swift

```diff
@@ -16,41 +16,27 @@ extension UIView {
     /// - parameter includeNonInteractable: Return views that do not have user interaction enabled. The default is false.
     ///
     /// - returns: An array of views, of type T, sorted by y position.
-    public func lookupSortedViews<T: UIView>(includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
-        if let textField = self as? T {
+    final public func sortedSubviews<T: UIView>(ofType type: T.Type, includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
```

Comment:
We need to document and explore the namespacing guidance on swift (via a blog post?!), but if the method is final, there's no objc-msgSend, so no selector, and no chance of clobbering.

If another library implemented the same method, there would be a conflict and you'd have to selectively import one or the other. This would be a painful, but would not be undefined behavior.